internal/kafka/internal/workers/clusters_mgr.go

```diff
@@ -1245,3 +1248,22 @@ func (c *ClusterManager) setKafkaPerClusterCountMetrics() error {
 
 	return nil
 }
+
+func (c *ClusterManager) setKasFleetManagerClusterProviderResourceQuotaConsumedMetric() error {
```

Comment:
Can we change the name to this? 
```suggestion
func (c *ClusterManager) setClusterProviderResourceQuotaMetrics() error {
```

I don't think we need separate methods betwen max allowed and consumed since both requires the same logic. The max allowed metric updated can be added in here as well.